hbase-server/src/main/java/org/apache/hadoop/hbase/wal/WALPrettyPrinter.java

```diff
@@ -79,9 +81,12 @@
   private boolean outputJSON;
   // The following enable filtering by sequence, region, and row, respectively
   private long sequence;
-  private String table;
+
+  // List of tables for filter
+  private Set<String> tableList;
```

Comment:
nit: for clarity, shouldn't we name it as _tableSet_ instead of _tableList_? Or maybe better to use List/ArrayList.

Reply:
+1, and this should be `final` too.

Reply:
I used Set since every WALEntry is going to search in the Set which is optimal for search. 
as @virajjasani  pointed out in another comment